src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -49,13 +49,18 @@ namespace
 namespace RenderTargetImpl
 {
 // Mutex to protect ID generation and our context-RenderTarget-map
-std::recursive_mutex mutex;
+std::recursive_mutex& getMutex()
+{
+    static std::recursive_mutex mutex;
+
+    return mutex;
```

Comment:
I wanted to say the same thing

Reply:
Newlines are always a coin toss for me... I have no personal preference really, but it would be nice to have a guideline somewhere since this isn't taken care of by tidy and not having to push newline corrections would save a lot of time.